package.json

```diff
@@ -68,13 +73,13 @@
     "coveralls": "2.11.9",
     "cross-env": "1.0.8",
     "dirsum": "0.1.1",
+    "electron-builder": "5.11.2",
     "electron-download": "2.1.2",
     "electron-ipc-mock": "0.0.2",
     "electron-mocha": "2.3.0",
-    "electron-packager": "7.2.0",
     "electron-rebuild": "1.1.5",
     "enzyme": "2.4.1",
-    "eslint": "3.0.0",
+    "eslint": "2.10.2",
```

Comment:
Is this eslint downgrade intentional? I thought the weird deps issue was fixed elsewhere?